kubernetes/zenko/values.yaml

```diff
@@ -12,18 +12,42 @@ ingress:
   # Used to create an Ingress record.
   # This must match the 'cloudserver' 'endpoint', unless your client
   # supports different hostnames.
-  hosts:
-    - zenko.local
+  # The root domain for endpoints
+  root_domain: zenko.local
+
+  # Subdomain to serve cloudserver traffic
+  # Will be added as a prefix to root_domain
+  # eg s3.example.com
+  s3_subdomain: s3
```

Comment:
If you leave `s3_subdomain` as an empty string would that still work and `zenko.local` remain as the default endpoint for S3? I am not talking about when both Blobserver and Cloudserver are deployed, but only when Blob is deployed.